### PR TITLE
Blokh/feat/seed collection flow

### PR DESCRIPTION
**Pull request by Blokh**
seed with kyb collection flow

### CODE REVIEW

File: services/workflows-service/scripts/workflows/ui-definition/pages/1-personal-info-page.ts [hunk 1]
```diff
@@ -11,7 +11,13 @@ const validationSchema = {
           properties: {
             additionalInfo: {
               type: 'object',
-              required: ['mainRepresentative'],
+              required: ['mainRepresentative', 'iHaveSigningAuthority'],
```
Comment (alonp99) [inline services/workflows-service/scripts/workflows/ui-definition/pages/1-personal-info-page.ts]: add translation keys to this file

File: services/workflows-service/scripts/workflows/ui-definition/pages/3-business-address-info-page.ts [hunk 2]
```diff
@@ -142,15 +83,11 @@ const validationSchema = {
...
-const physicalAddressForm = {
-  '==': [{ var: 'entity.data.additionalInfo.headquarters.isDifferentFromPhysical' }, true],
-};
-
 export const BusinessAddressInfoPage = {
   type: 'page',
   number: 3,
   stateName: 'business_address_information',
-  name: 'text.businessAddress',
+  name: 'Headquarters Address',
```
Comment (alonp99) [inline services/workflows-service/scripts/workflows/ui-definition/pages/3-business-address-info-page.ts]: restore key

File: services/workflows-service/scripts/workflows/ui-definition/pages/5-company-documents.ts [hunk 3]
```diff
@@ -0,0 +1,268 @@
+const validationSchema = [
+  {
+    documentId: 'document-bank-statement',
+    destination: 'pages[0].ballerineFileId',
+    required: true,
+    errorMessage: 'This field is required',
+  },
```
Comment (alonp99) [inline services/workflows-service/scripts/workflows/ui-definition/pages/5-company-documents.ts]: translation

**Review by alonp99 — APPROVED** [review]

File: services/workflows-service/src/workflow/workflow.service.ts [hunk 4]
```diff
@@ -75,6 +75,7 @@ import {
   WorkflowRuntimeDataRepository,
 } from './workflow-runtime-data.repository';
 import mime from 'mime';
+import { env } from '@/env';
```
Comment (Omri-Levy) [inline services/workflows-service/src/workflow/workflow.service.ts:83]: To my understanding `configService` now uses `env` under the hood. See if you can get `configService` to work here.

**Pull request merged**